src/command/circle.ts

```diff
@@ -1,140 +0,0 @@
-import Command, { CommandContext } from "../api/command";
-import Bot from "../api/bot";
-import { CategoryChannel, Message, OverwriteResolvable } from "discord.js";
-import Wizard, {
-  ColorWizardNode,
-  EmojiWizardNode,
-  GraphicWizardNode,
-  TextWizardNode,
-  UserMentionWizardNode,
-} from "../util/wizard";
-import { settings } from "../settings";
-import { CircleData } from "../api/schema";
-
-export default class CircleCommand extends Command {
-  constructor() {
-    super({
-      name: "circle",
-      description: "A suite of command that manage ACM Community Circles.",
-      dmWorks: false,
-      userPermissions: BigInt(268443664),
-    });
-  }
-
-  public async exec({ msg, bot, args }: CommandContext) {
-    switch (args[0]) {
-      case "add":
-        await addCircle(bot, msg, args);
-        break;
-      case "repost":
```

Comment:
slash command is missing repost functionality, so well never be able to see the new circle data nor update existing circle data